proxy/coco/tests/working_copy.rs

```diff
@@ -0,0 +1,87 @@
+use coco::{project::checkout, state, RunConfig};
+
+use assert_matches::assert_matches;
+use pretty_assertions::assert_eq;
+
+#[macro_use]
+mod common;
+use common::{build_peer, init_logging, shia_le_pathbuf};
+
+#[tokio::test]
+async fn upstream_for_default() -> Result<(), Box<dyn std::error::Error>> {
+    init_logging();
+
+    let alice_tmp_dir = tempfile::tempdir()?;
+    let alice_repo_path = alice_tmp_dir.path().join("radicle");
+    let alice_peer = build_peer(&alice_tmp_dir, RunConfig::default()).await?;
+    let alice = state::init_owner(&alice_peer.peer, "alice".to_string()).await?;
+
+    let alice_peer = {
+        let peer = alice_peer.peer.clone();
+        tokio::task::spawn(alice_peer.into_running());
+        peer
+    };
+
+    let _ = state::init_project(
+        &alice_peer,
+        &alice,
+        shia_le_pathbuf(alice_repo_path.clone()),
+    )
+    .await?;
+
+    let repo = git2::Repository::open(alice_repo_path.join("just"))
+        .map_err(radicle_surf::vcs::git::error::Error::from)?;
```

Comment:
This has been confusing to me in existing tests, too. `alice_repo_path` is not the repo path but the path that the repo is created in. We could just use `alice_tmp_dir.path()` for that. It’s also a bit unclear where the `just` comes from. Maybe it would make sense to extract it from the return value of `shia_le_pathbuf` or excplicitly construct `project::Create` here.